tests/Api/Admin/AdminUsersTest.php

```diff
@@ -41,4 +40,25 @@ public function it_allows_admin_users_to_log_in(): void
 
         $this->assertResponse($response, 'admin/log_in_admin_response', Response::HTTP_OK);
     }
+
+    public function it_sends_administrator_password_reset_email(): void
+    {
+        $this->loadFixturesFromFile('authentication/api_administrator.yaml');
+
+        $this->client->request(
+            Request::METHOD_POST,
+            '/api/v2/admin/reset-password-requests',
+            [],
+            [],
+            self::CONTENT_TYPE_HEADER,
+            json_encode([
+                'email' => 'api@example.com',
+                'locale' => 'pl_PL',
+            ])
+        );
+
+        $response = $this->client->getResponse();
+
+        $this->assertResponse($response, 'admin/reset_password_request_response', Response::HTTP_CREATED);
```

Comment:
The expected response file is not created, as far as I see? The test is also not run, because it does not have the `@test` annotation added 🔥